packages/kbn-pm/src/run.ts

```diff
@@ -41,7 +46,46 @@ export async function runCommand(command: ICommand, config: Omit<ICommandConfig,
       ...config,
       kbn,
     });
+
+    if (command.reportTiming) {
+      const reporter = CiStatsReporter.fromEnv(log);
+      await reporter.timings({
+        upstreamBranch: kbn.kibanaProject.json.branch,
```

Comment:
What was the reason for fetching the branch and UUID here instead of CIStats handling it

Reply:
`@kbn/pm` is built into the repo and we can't rely on other code being built when it runs, so it felt logical to avoid relying on `@kbn/utils` or anything like that, and I also didn't want to explode the size of the `@kbn/pm` dist either.